### ISSUE TITLE
Wrong chart template for Trafik IngressRoute

### ISSUE THREAD

**Issue by vmarchese** [k8ssandra/k8ssandra#61]
## Bug Report

**Describe the bug**
Enabling the cassandra Traefik TCP Route in the yaml values file for the k8ssandra cluster gives an error in the chart yaml:

```
Error: YAML parse error on k8ssandra-cluster/templates/traefik.ingressroutes.yaml: error converting YAML to JSON: yaml: line 5: mapping values are not allowed in this contex
```

**To Reproduce**

Prepare a k8ssandra-cluster yaml values file as in:

```yaml
ingress:
  traefik:
    enabled: true
    cassandra:
      enabled: true
      entrypoints: 
        - cassandra
```

Try to install (or dry-run) the chart with:

```bash
> helm install k8ssandra-cluster-a ./k8ssandra-cluster   -f kvalues.yaml --dry-run
Error: YAML parse error on k8ssandra-cluster/templates/traefik.ingressroutes.yaml: error converting YAML to JSON: yaml: line 5: mapping values are not allowed in this context
```

**Proposed Resolution**
The file `k8ssandra-cluster/templates/traefik.ingressroutes.yaml` is missing a `label` stanza in the ingressroute object (line 33) :

```yaml
---
apiVersion: traefik.containo.us/v1alpha1
kind: IngressRouteTCP
metadata:
  name: {{ $releaseName }}-k8ssandra-cassandra
{{ include "k8ssandra-cluster.labels" . | indent 4 }}
spec:
  entryPoints:
...
```

It should be:
```yaml
---
apiVersion: traefik.containo.us/v1alpha1
kind: IngressRouteTCP
metadata:
  name: {{ $releaseName }}-k8ssandra-cassandra
  labels:
{{ include "k8ssandra-cluster.labels" . | indent 4 }}
spec:
  entryPoints:
...
```

**Comment by jsanda** [comment]
Hi @vmarchese,

Nice find! Would you like to submit a PR with the fix?

**Comment by vmarchese** [comment]
> Hi @vmarchese,
> 
> Nice find! Would you like to submit a PR with the fix?

Yep, why not? :)

**Comment by jsanda** [comment]
Fixed by #62
